packages/vitest/src/node/execute.ts

```diff
@@ -97,13 +130,13 @@ export class VitestRunner extends ViteNodeRunner {
       __vite_ssr_dynamic_import__: requestWithMock,
 
       // vitest.mock API
-      __vitest__mock__: this.mocker.mockPath,
-      __vitest__unmock__: this.mocker.unmockPath,
-      __vitest__importActual__: importActual,
-      __vitest__importMock__: importMock,
+      __vitest_mock__: this.mocker.queueMock.bind(this.mocker),
+      __vitest_unmock__: this.mocker.queueUnmock.bind(this.mocker),
+      __vitest_importActual__: importActual,
+      __vitest_importMock__: importMock,
       // spies from 'jest-mock' are different inside suites and execute,
       // so wee need to call this twice - inside suite and here
-      __vitest__clearMocks__: this.mocker.clearMocks,
+      __vitest_clearMocks__: this.mocker.clearMocks.bind(this.mocker),
```

Comment:
Instead of injecting multiple apis, do you think we should inject a context object (or the mocker directly) so we could also provide better type-safty?

Reply:
I was thinking the same, when I had to put 9 `@ts-expect-error` in `vi` object 👀

Reply:
I've moved eveyrthing mock related to `VitestMocker` now